bertopic/_bertopic.py

```diff
@@ -928,12 +929,14 @@ def hierarchical_topics(self,
 
         Arguments:
             docs: The documents you used when calling either `fit` or `fit_transform`
+            use_ctfidf: Whether to calculate distances between topics based on c-TF-IDF embeddings. If False, semantic
+                        embeddings are used.
```

Comment:
Can you rephrase *semantic embedding* to something like *If False, the embeddings from the embedding model are used*? The input does not have to be semantic embeddings and can technically also be the input from Bag-of-Words/TF-IDF.

Reply:
Sure. Done